linkerd/cache/src/lib.rs

```diff
@@ -1,113 +1,116 @@
 #![deny(warnings, rust_2018_idioms)]
 
-use self::cache::Cache;
-pub use self::layer::Layer;
-pub use self::purge::Purge;
 use futures::{future, Async, Poll};
-use linkerd2_error::Error;
+use indexmap::IndexMap;
```

Comment:
do we actually use `IndexMap`'s ordered property here? AFAICT, we don't, so it might be better to just use `HashMap`...